data/incipias/incipias first contact.txt

```diff
@@ -0,0 +1,405 @@
+	to complete
+		never
+	on enter Dom'us
+		conversation
+			`As you enter the system, you notice ships of a type you have never seen before. Hailing them only gives you static. After some more unsuccessful attempts, you give up. Maybe you can find some more information in nearby systems.`
+				to display
+					not "Visit Quarg in Incipias Space: offered"
+			`As you enter the system, you notice ships of a type you have never seen before. Hailing them only gives you static. After some more unsuccessful attempts, you give up. This must be the species the Quarg talked about.`
+				to display
+					has "Visit Quarg in Incipias Space: offered"
+
+
+
+mission "Visit Quarg in Incipias Space"
+	invisible
+	landing
+	source
+		government "Quarg (Incipias)"
+	to complete
+		never
+	on offer
+		conversation
+			branch "ut divitas" "spec inci"
+				has "flagship planet: Ut Divitas"
+			label "ut divitas"
+			`Your spaceship lands with a soft jolt. A small group of Quarg quickly gathers around your ship, making some simple but clear gestures telling you to wait. After they've realized that you understood, one of them walks away in a hurried manner.`
+			`	You have to wait for quite some time until a different Quarg arrives, presumably one that can speak your language. It begins to speak in an accent you can't place.`
+				goto "both planets"
+			label "spec inci"
+			`	You have to wait for a few hours until a group of three Quarg appear. The one in the middle begins to speak in an accent you can't place.`
+			label "both planets"
+			`	"Welcome, human. Your lengthy wanderings have brought forth you here, yet your kind has never set foot in this region of the stars. May we inquire as to your mission?"`
+			choice
+				`	"I am just exploring the galaxy."`
+					goto explorer
+				`	"I am looking for new species."`
+					goto "new species"
+			label explorer
+			`	"It is an obvious pursuit for the young," it nods. "If I may advise you, do not perambulate across this quarter of space in a careless manner. The galaxy carries many perilous situations that your kind has yet to uncover or even to comprehend." It straightens up to its full height, towering over you.`
+				goto end
+			label "new species"
+			`	"What a peculiar task that is." It looks confused and tilts its head. "What do you hope to achieve by this? That an unfathomable amount of dangers come with encountering and treating with other species, did you not foresee? The boundaries of space are not idle accidents. I advise you to be scrupulous and ponder every last step."`
+			label end
+			`	After a short pause it continues talking; you notice an almost-hidden frustration lurking beneath the Quarg's face. "You are authorized to continue your nomadic voyagings here, but there are demands we Quarg make of you. Within a trifling interval of this moon another kind has initiated their journey to the stars. Do not interrupt them for your own sake, as we will avenge any hostile actions against them."`
```

Comment:
I was the one that added 'voyagings', to suggest that this Quarg doesn't have a very good grasp of humanese.

Reply:
I think most people will understand what the word is supposed to mean?

Reply:
Fair enough. Changed voyagings back; the rest of the suggestion still applies.

Reply:
I mean, if one is reading the conversation, then they'd see my comment later saying it's okay; it would be odd to just delete something in a prior comment when other comments exist based on that discussion.